lib/std/net.zig

```diff
@@ -437,7 +437,7 @@ pub fn getAddressList(allocator: *mem.Allocator, name: []const u8, port: u16) !*
         const name_c = try std.cstr.addNullByte(allocator, name);
         defer allocator.free(name_c);
 
-        const port_c = try std.fmt.allocPrint(allocator, "{}\x00", .{port});
+        const port_c = try std.fmt.allocPrintCstr(allocator, "{}", .{port});
```

Comment:
I liked the old code here. I don't think `allocPrintCstr` should really get its own function.

Reply:
That's fair, I wasn't sure about it either